modules/devices/e2k/processor.c

```diff
@@ -272,6 +298,10 @@ GSList *processor_scan(void)
         }
 
         if (processor) {
+            /* set full model name*/
+            if(g_str_has_prefix(tmp[0], "model name"))
+                tmp[1] = g_strdup(processor_get_full_name(tmp[1]));
```

Comment:
```suggestion
                tmp[1] = processor_get_full_name(tmp[1]);
```

Reply:
I have fixed your comments except for this one. When fixing this comment, an error occurs `munmap_chunk(): invalid pointer`